calico_node/allocateipip/allocate_ipip_addr.go

```diff
@@ -2,13 +2,20 @@ package main
 
 import (
 	"os"
+	"time"
 
 	log "github.com/sirupsen/logrus"
 
+	"context"
```

Comment:
nit:  i think the context import should live with the other native imports at the top.